docs/_markbind/variables.md

```diff
@@ -1,7 +1,9 @@
-<span id="showBaseUrl">
+<span id="showBaseUrlCodeStyle">
 <code>{<span></span>{ baseUrl }}</code>
 </span>
 
+<span id="showBaseUrl">{<span></span>{ baseUrl }}</span>
```

Comment:
How about `<code>{<span></span>{ baseUrl }}</code>` -> `{{ showBaseUrlCode }}`

and `{<span></span>{ baseUrl }}` -> `{{ showBaseUrlText }}`